docs/arby/README.md

```diff
@@ -1,4 +1,10 @@
 # Arby
 Arby looks for arbitrage opportunities on the OpenDEX network. Once a profitable trade appears it will automatically attempt to enter it. It will also enter active orders to the orderbook and updates them every 1 minute by default.
 
+Currently, it is possible to use Arby for arbitrage trades between OpenDEX and Binance for LTCBTC trading pair. The system is designed in a way that allows more exchanges and trading pairs to be added in the future.
+
 [Setup Guide](./setup.md)
+
+## How does it work?
```

Comment:
Diagram comments:
- OpenDEX Taker (TBD) will be the Hedgy functionality I take it?
- `VOLATILITY_TIMEOUT` is https://github.com/ExchangeUnion/market-maker-tools/issues/4, right? Already implemented?
- market buy/sell order fallback: I think we should never do market orders, rather issue a new limit order if not filled within `LIMIT_ORDER_TIMEOUT`

Reply:
> OpenDEX Taker (TBD) will be the Hedgy functionality I take it?

Not exactly, hedgy will rebalance regardless of the price. This is for the scenario where there is a better price on OpenDEX (highly unlikely in the short term) than on centralized exchange.

> market buy/sell order fallback: I think we should never do market orders, rather issue a new limit order if not filled within LIMIT_ORDER_TIMEOUT

If the market becomes volatile while waiting for the asymmetric trade it could cause huge losses. What happens if the second limit order does not succeed? Third? Forth? Not sure if 100% excluding market orders is the way to go.

Reply:
> VOLATILITY_TIMEOUT is #4, right? Already implemented?

Not yet (as mentioned in the OP this describes the ideal logic and does not reflect the current logic 100%). 

I wanted to clarify what the logic should be when "returning" from the break/timeout period. What is the criteria to ensure that the market is no longer volatile? Price movement X% in Y% of time?

Reply:
> If the market becomes volatile while waiting for the asymmetric trade it could cause huge losses. What happens if the second limit order does not succeed? Third? Forth? Not sure if 100% excluding market orders is the way to go.

Moved this to a separate issue: https://github.com/ExchangeUnion/market-maker-tools/issues/7

Reply:
> > VOLATILITY_TIMEOUT is #4, right? Already implemented?
> 
> Not yet (as mentioned in the OP this describes the ideal logic and does not reflect the current logic 100%).
> 
> I wanted to clarify what the logic should be when "returning" from the break/timeout period. What is the criteria to ensure that the market is no longer volatile? Price movement X% in Y% of time?

Also moved to separate issue: https://github.com/ExchangeUnion/market-maker-tools/issues/8